orchestrator/careplancontributor/service.go

```diff
@@ -524,12 +533,41 @@ func (s Service) handleGetContext(response http.ResponseWriter, _ *http.Request,
 
 func (s Service) withSessionOrBearerToken(next func(response http.ResponseWriter, request *http.Request)) http.HandlerFunc {
```

Comment:
can you rename this to something that resembles its intended consumers better? Something maybe `withUserAuth` (also ugly, but might be clearer? `user` also being an API client..)